implementations/rust/ockam/ockam_command/src/logs/mod.rs

```diff
@@ -24,20 +23,16 @@ fn log_max_files() -> usize {
     get_env_with_default("OCKAM_LOG_MAX_FILES", default).unwrap_or(default) as usize
 }
 
+fn log_format() -> String {
+    let default = "full".to_string();
```

Comment:
So it can actually be `json`, `pretty` or `full`? Why not use an enum to model those 3 cases exactly?

Reply:
I'm actually going to use "default" instead of "full" for that case. On it!